model_db.py

```diff
@@ -1,10 +1,12 @@
 import collections
 import logging
 import os
+from uuid import uuid4
```

Comment:
Is there a difference between `uuid4` and another `uuid` package we import elsewhere in the program? I think the other one is just `import UUID`?

Reply:
we have:
* `from uuid import UUID` for the class
* `from uuid import uuid4` for one of the (many) "constructors"

Reply:
Ah ok, so different use cases, I was worried about slightly different functionality. But then again, there's not much variance a uuid can have, so I doubt it would cause an issue